### ISSUE TITLE
cannot connect to facebook

### ISSUE THREAD

**Issue by swinokur** [miranda-ng/miranda-ng#4167]
### Expected behavior
My facebook login (chrome, desktop) and Facebook messenger lite login (android) are both working. I am also having connection issues with Pigidn IM's facebook plugins. I was hoping that Miranda would work.

### Actual behavior
Miranda NG is not connecting to facebook and giving "unknown errors"

### Does it work in stable version?
The login does not work in either the stable or dev version.

### Steps to reproduce that behavior
 * Provide step-by-step instructions so we can reproduce the issue (We can't fix what we can't reproduce).
in a fresh empty Miranda install, I added only my facebook account and tried to connect. Miranda did not connect and returned an error. Unfortunately the error / error-code seems to be non-specific.

 * Attach your VersionInfo.

> CPU: Intel(R) Core(TM) i7-6950X CPU @ 3.00GHz [Intel64 Family 6 Model 79 Stepping 1] [DEP Enabled] [20 CPUs]
> Installed RAM: 65446 MBytes
> Microsoft Windows 7 Enterprise Edition, 64-bit Service Pack 1 (build 7601)
> Internet Explorer: 9.11.9600.17843 (build 99600)
> Administrator privileges: Yes
> OS Languages: (UI | Locale (User/System)) : English/English | English/English
> Free disk space on Miranda partition: 132239 MBytes
> 
> Miranda NG Version: 0.96.5 alpha build #26802 (b31af4096a) x64
> Build time: 3 Feb 2024 4:34:52
> Profile: C:\Users\[...]\AppData\Roaming\Miranda NG\default\default.dat
> Database driver: SQLite database driver
> Profile size: 110592 Bytes
> Profile creation date: 2 Feb 2024 22:33:20
> Language pack: No language pack installed
> Service Mode: No
> SRMM Log Viewer: built-in
> 
> **Active Plugins (17):**
> ¤ AVS.dll v.[b]0.98.1.6[/b] [3 Feb 2024 4:34:56] - Avatar service 
> ¤ Clist_modern.dll v.[b]0.9.1.12[/b] [3 Feb 2024 4:35:37] - Modern contact list 
> ¤ CrashDumper.dll v.[b]0.2.0.2[/b] [3 Feb 2024 4:35:40] - Crash dumper 
> ¤ Db_autobackups.dll v.[b]0.1.0.7[/b] [3 Feb 2024 4:35:09] - Db autobackuper 
> ¤ DbChecker.dll v.[b]0.96.4.1[/b] [3 Feb 2024 4:37:25] - Database checker 
> ¤ Dbx_sqlite.dll v.[b]0.96.4.1[/b] [3 Feb 2024 4:37:18] - Miranda NG SQLite database driver 
> ¤ Dummy.dll v.[b]0.2.0.1[/b] [3 Feb 2024 4:36:40] - Dummy protocol 
> ¤ Facebook.dll v.[b]0.1.0.11[/b] [3 Feb 2024 4:37:23] - Facebook 
> ¤ ICQ.dll v.[b]0.96.5.1[/b] [3 Feb 2024 4:37:18] - ICQ-WIM protocol 
> ¤ Import.dll v.[b]0.96.3.1[/b] [3 Feb 2024 4:35:31] - Import contacts and messages 
> ¤ IRC.dll v.[b]0.96.1.4[/b] [3 Feb 2024 4:34:57] - IRC protocol 
> ¤ Jabber.dll v.[b]0.95.11.1[/b] [3 Feb 2024 4:35:50] - Jabber protocol 
> ¤ NewEventNotify.dll v.[b]0.2.2.7[/b] [3 Feb 2024 4:35:35] - NewEventNotify 
> ¤ PluginUpdater.dll v.[b]0.2.1.7[/b] [3 Feb 2024 4:35:53] - Plugin updater 
> ¤ StatusManager.dll v.[b]1.2.0.6[/b] [3 Feb 2024 4:37:08] - Status manager 
> ¤ TabSRMM.dll v.[b]3.6.2.6[/b] [3 Feb 2024 4:35:06] - TabSRMM 
> ¤ TopToolBar.dll v.[b]0.8.0.5[/b] [3 Feb 2024 4:35:29] - TopToolBar 
> 
> **Protocols and Accounts:**
> -------------------------------------------------------------------------------
> Facebook                 1 - Enabled 0 - Disabled  Loaded
> 
> **Icon Packs:**
> -------------------------------------------------------------------------------
>  Proto_Dummy.dll [3 Feb 2024 4:41:50]
>  Proto_Facebook.dll [3 Feb 2024 4:41:51]
>  Proto_ICQ.dll [3 Feb 2024 4:41:51]
>  Proto_IRC.dll [3 Feb 2024 4:41:49]
>  Proto_Jabber.dll [3 Feb 2024 4:41:49]
>  Proto_MetaContacts.dll [3 Feb 2024 4:41:50]
>  Proto_MRA.dll [3 Feb 2024 4:41:51]
>  TabSRMM_icons.dll [3 Feb 2024 4:41:50]
>  Toolbar_icons.dll [3 Feb 2024 4:41:49]
>  xStatus_Jabber.dll [3 Feb 2024 4:41:50]

[netlog.txt](https://github.com/miranda-ng/miranda-ng/files/14154237/netlog.txt)

Thanks for looking! Please let me know if there are other tests I can run or things to try.

**Comment by georgehazan** [comment]
Looks like smth blocks the access, because the request to FB finishes without any result, neither positive nor negative.
Or Facebook stopped answering this way? But for now you're the only one with this problem

**Comment by georgehazan** [comment]
Another users confirmed that they can log in FB

**Comment by swinokur** [comment]
Thanks for looking into this. I still cannot connect using either Pidgin or Miranda. I did try using a VPN with Miranda on another machine to see if things were different if I tried to connect with VPN endpoints in UK, Canada, Japan, but saw the same behavior. 

I've also tried changing my FB password in case there was something amok with that. 

Is there a way to get Miranda to log even more verbosely than above? 
thanks!
